meson.build

```diff
@@ -38,6 +38,16 @@ if cpp.get_id() == 'msvc'
   add_project_arguments(msvc_args, language : 'cpp')
   # Disable SAFESEH with MSVC for libs that use external deps that are built with MinGW
   # noseh_link_args = ['/SAFESEH:NO']
+else
+  add_global_arguments('-fno-rtti', language : 'cpp')
+  add_global_arguments('-fno-exceptions', language : 'cpp')
+  add_global_arguments('-fno-threadsafe-statics', language : 'cpp')
```

Comment:
In `configure.ac` this looks conditional based on having gcc as compiler.

In meson you can do that with `if cpp.get_id() == 'gcc'` (table here: https://mesonbuild.com/Reference-tables.html#compiler-ids) or (perhaps) better don't hard-code a specific compiler but check if the arguments are supported, e.g. like this:
```
  add_global_arguments(cpp.get_supported_arguments(['-fno-rtti', '-fno-exceptions', ...]), language: 'cpp')
```

Reply:
oh, ok, as is needed for clang also, clang impersonate gcc in autotools apparently, how should I check that here?

Reply:
I think you should just use `cpp.get_supported_arguments()` or individual `if cpp.has_argument(xyz)` if you want to do it one-by-one. Without special-casing any compilers. If that doesn't work, we can fix it once we know the exact failure.

Reply:
Oh ok.